CrossSectionAnalysis/Resources/UI/CrossSectionAnalysis.ui

```diff
@@ -612,13 +622,41 @@ The input centerline is expected to be inside the lumen surface.</string>
            </property>
           </widget>
          </item>
-         <item row="1" column="0">
-          <widget class="QLabel" name="label_2">
+         <item row="2" column="0">
+          <widget class="QLabel" name="mirrorLabel">
            <property name="text">
-            <string>Long.:</string>
+            <string>Mirror:</string>
            </property>
           </widget>
          </item>
+         <item row="2" column="1">
+          <layout class="QHBoxLayout" name="mirrorHorizontalLayout">
+           <item>
+            <widget class="QCheckBox" name="mirrorXCheckBox">
+             <property name="toolTip">
+              <string>Rotate by 180° around the slice view's X-axis.

```

Comment:
mirrorX would be 180deg rotation around the Y axis. Anyway, use the horizontal and vertical flip terms instead.